llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -346,8 +346,9 @@ bool sampledTypeIsSignedInteger(const llvm::Type *HandleType) {
   if (TET->getTargetExtName() == "spirv.Image") {
     return false;
   }
-  return TET->getTypeParameter(0)->isIntegerTy();
-}
+  assert(TET->getTargetExtName() == "spirv.SignedImage") {
+    return TET->getTypeParameter(0)->isIntegerTy();
+  }
```

Comment:
braces after the assert?

Reply:
I pushed before I was support to. This does not even compile.